http/headers/set-cookie.json

```diff
@@ -172,12 +172,26 @@
               "opera_android": {
                 "version_added": "41"
               },
-              "safari": {
-                "version_added": "12"
-              },
-              "safari_ios": {
-                "version_added": "12.2"
-              },
+              "safari": [
+                {
+                  "version_added": "13"
+                },
+                {
+                  "version_added": "12",
+                  "partial_implementation": true,
+                  "notes": "Treats <code>SameSite=None</code> and invalid values as <code>Strict</code> in macOS before 10.15 Catalina. See <a href='https://webkit.org/b/198181'>bug 198181</a>."
```

Comment:
this note actually also applies to Safari 13 (as it is possible to install safari 13 on macOS 10.14 Mojave, but the fix is done by the OS upgrade, not by the Safari upgrade alone)

Reply:
Added a note for Safari 13 as well, but no partial_support since there is full support if you are on the latest OS.